packages/js/src/workouts/components/SocialInputPersonSection.js

```diff
@@ -39,7 +39,7 @@ function SocialInputPersonSection( { personId } ) {
 					</Alert>
 					<p>
 						{
-							createInterpolateElement(
+							addLinkToString(
 								sprintf(
 									__(
 										// translators: %1$s and %2$s are replaced by opening and closing anchor tags
```

Comment:
this translator comment is placed erroneously: it should be placed on the line directly preceding the `__(` function call

Reply:
Same goes for line #62 so please check elsewhere